documentation/install_from_packages.markdown

```diff
@@ -1,16 +1,27 @@
 ---
 layout: default
-title: "Puppet Server: Installing From Packages"
+title: "Install Puppet Server"
 canonical: "/puppetserver/latest/install_from_packages.html"
 ---
 
-[repodocs]: https://puppet.com/docs/puppet/latest/puppet_platform.html
+Puppet Server is a required application that runs on the Java Virtual Machine (JVM). It performs the role of the master node that controls configuration information for one or more managed agent nodes
 
-Puppet Server is configured to use 2 GB of RAM by default. If you're just testing an installation on a Virtual Machine, this much memory is not necessary. To change the memory allocation, see [Memory Allocation](#memory-allocation).
+> Note: If you have any issues with the steps below, submit these to our [bug tracker](https://tickets.puppet.com/browse/SERVER).
 
-> If you're also using PuppetDB, check its [requirements](https://puppet.com/docs/puppetdb/latest/index.html#system-requirements).
+## Before you begin
 
-## Java support
+Review the system requirements and make sure you have a supported version of Java. 
+
+### Supported operating systems
+
+Puppet provides official packages that install Puppet Server 6 and all of its prerequisites on x86_64 architectures for the following platforms:
+
+* Red Hat Enterprise Linux 6, 7
+* Debian 8 (Jessie), 9 (Stretch), 10 (Buster)
+* Ubuntu 18.04 (Bionic), 16.04 (Xenial)
```

Comment:
nitpick, but Red Hat and Debian have older versions listed first while Ubuntu lists newest first - I think we should be consistent.